docs/tutorials/getting_started/moran.rst

```diff
@@ -132,3 +132,18 @@ Other types of implemented Moran processes:
 
 - :ref:`moran-process-on-graphs`
 - :ref:`approximate-moran-process`
+
+Atomic Mutation for Evolvable Players
```

Comment:
I wonder if this should perhaps not be in the `getting_started` section? Perhaps just include the code as an example for what you've written in the `further_topics` section? (If you feel strongly about having it here that's fine.)

Reply:
I don't feel strongly about it but there's also not a lot to write about other than the Moran process until we move the dojo algorithms over, in which case the ideal documentation structure should be more clear.

Reply:
I thought it about it some more and I think if you're ok with it I'd like us to move it to `further_topics`. I think in the `getting_started` section it might be a bit confusing for newcomers to game theory.

Reply:
Done